### DISCUSSION TITLE
The 4th page of my GitHub Pages site won't load

### DISCUSSION THREAD

**Discussion by HazardousFDM** [community/community#21663] [Other Feature Feedback, Questions, & Ideas]
<p>Hello all!,<br>
Im teaching myself html/css (2 weeks in) and I made a dumb site to post on GitHub pages for fun.<br>
Everything was fine. Every page loaded (wellcome-pg5) and looked good.<br>
I was making updates from atom editor and all of a sudden…, the 4th page got a 404 error.<br>
Ive looked for 3 days on a solution with no avail.<br>
Now on day 4. Ive ran all my code trough Linters for error checks with ZERO! (proud of that!)<br>
Does anyone have some insight into this problem? all pages load except one?<br>
Thank you all!</p>
<p>PS: I have a feeling it has something to do with my html coding and the linter didn’t catch it or see it &amp; I haven’t noticed it. HELP! lol</p>

**Comment by jfullstackdev** [comment]
<p>can you please give the link for the repo of your site, so that we can check that,</p>

**Comment by HazardousFDM** [comment]
<p>Sorry about that. Here you go.<br>
<a href="https://hazardousfdm.github.io/TheHazardServer/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://hazardousfdm.github.io/TheHazardServer/</a><br>
Its a dumb site. first one I’ve ever made.</p>

**Comment by jfullstackdev** [comment]
<p>and the other one that is not working, you provided the site, not the repo</p>
<p>please include the repo also</p>

**Comment by HazardousFDM** [comment]
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/HazardousFDM/TheHazardServer" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/81825468/181066076-19c73fc6-9738-41a6-9aba-6dfb783b72e0.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/9/19956487e1dd7584f8ba573e3729916746766734_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/9/19956487e1dd7584f8ba573e3729916746766734_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/9/19956487e1dd7584f8ba573e3729916746766734.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/9/19956487e1dd7584f8ba573e3729916746766734_2_10x10.png"></div>

<h3><a href="https://github.com/HazardousFDM/TheHazardServer" target="_blank" rel="noopener nofollow ugc">HazardousFDM/TheHazardServer</a></h3>


  <p><span class="label1">A web guide on install and setup of a Plex Media Server and how to obtain content.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by jfullstackdev** [comment]
<p>what filename of the page is not displaying as page on GitHub Pages?</p>

**Comment by HazardousFDM** [comment]
<p>“KidsTVShows.html” is the page that won’t load.</p>

**Comment by jfullstackdev** [comment]
<p>and also, what I noticed is that, you will find it difficult, please separate each page as separate folder</p>

**Comment by HazardousFDM** [comment]
<p>o, I never thought about doing that. good idea. ill give that a try from now on.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p><a href="https://hazardousfdm.github.io/TheHazardServer/KidsTvShows.html" class="onebox" target="_blank" rel="noopener nofollow ugc">https://hazardousfdm.github.io/TheHazardServer/KidsTvShows.html</a></p>
<p>the problem is the URL</p>
<p><a href="https://hazardousfdm.github.io/TheHazardServer/KidsTVShows.html" class="onebox" target="_blank" rel="noopener nofollow ugc">https://hazardousfdm.github.io/TheHazardServer/KidsTVShows.html</a></p>
<p>this last one is working</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/7/1745dac1bd44dd49341cb7b928a64f55bbc817e9.jpeg" data-download-href="/uploads/short-url/3jSDXdfKczvspwStiOmeJOwxUcF.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181066078-5b88cf46-be62-4b1c-ac9d-a6c1c430d9af.jpeg" alt="image" data-base62-sha1="3jSDXdfKczvspwStiOmeJOwxUcF" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/1745dac1bd44dd49341cb7b928a64f55bbc817e9_2_690x387.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/7/1745dac1bd44dd49341cb7b928a64f55bbc817e9_2_1035x580.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/7/1745dac1bd44dd49341cb7b928a64f55bbc817e9.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/1745dac1bd44dd49341cb7b928a64f55bbc817e9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1366×768 141 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by HazardousFDM** [comment]
<p>ok. I see it now! omg…lol Thank you so much. I knew it was something dumb that I just couldn’t notice.</p>
